src/routes/admin/users/+page.server.js

```diff
@@ -1,5 +1,6 @@
 import {getUserRoles} from "$lib/server/account.js";
 import {redirect} from "@sveltejs/kit";
+import {getUsers} from "../../../lib/server/users.js";
```

Comment:
Cet import peut être raccourci.
```suggestion
import {getUsers} from '$lib/server/users.js';
```